cirq-core/cirq/study/resolver_test.py

```diff
@@ -244,9 +244,11 @@ def _resolved_value_(self):
     c = sympy.Symbol('c')
     r = cirq.ParamResolver({a: foo, b: bar, c: baz})
     assert r.value_of(a) is foo
-    assert r.value_of(b) is b
     assert r.value_of(c) == 'Baz'
 
+    with pytest.raises(ValueError, match='Could not resolve parameter b'):
+        _ = r.value_of(b)
```

Comment:
I think we should move this to a separate test function marked with a strict xfail, for example,

```python
@pytest.mark.xfail(reason='this test requires sympy 1.12', strict=True)
def test_custom_value_not_implemented():
    ...
```

When the new sympy is released, the strict xfail will produce a CI failure 
after which we can remove the xfail mark.

Reply:
Good idea done.